packages/extension-chakra-storefront/src/theme/components/base/input.js

```diff
@@ -1,54 +1,48 @@
 /*
- * Copyright (c) 2021, salesforce.com, inc.
+ * Copyright (c) 2025, salesforce.com, inc.
  * All rights reserved.
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import {defineRecipe} from '@chakra-ui/react'
 
-const mdSize = {height: 11, borderRadius: 'sm'}
-
 export default defineRecipe({
-    base: {
-        field: {
-            _focus: {
-                borderColor: 'blue.600'
-            }
-        }
-    },
+    base: {},
     variants: {
         variant: {
             outline: {
                 field: {
                     borderColor: 'gray.500'
                 }
             },
+            // Note: Chakra v3 input has removed filled variant
+            // we added it back to preserve the style in the template
             filled: {
-                // we use filled variant for
-                // search input
-                field: {
-                    borderColor: 'gray.600',
+                border: '2px solid {colors.gray.600}',
+                backgroundColor: 'gray.100',
+                focusVisibleRing: 'inside',
+                focusRingColor: 'var(--focus-color)',
+                _focus: {
+                    backgroundColor: 'white'
+                },
+                _focusVisible: {
+                    background: 'transparent',
+                    borderColor: 'blue.500',
+                    outlineWidth: 0
+                },
+                _hover: {
                     backgroundColor: 'gray.100',
                     _focus: {
                         backgroundColor: 'white'
-                    },
-                    _hover: {
-                        backgroundColor: 'gray.100',
-                        _focus: {
-                            backgroundColor: 'white'
-                        }
-                    },
-                    _placeholder: {
-                        color: 'gray.700'
                     }
+                },
+                _placeholder: {
+                    color: 'gray.700'
                 }
             }
-        },
-        size: {
-            md: {
-                ...mdSize,
-                px: 3
-            }
         }
+    },
+    defaultVariants: {
+        size: 'lg'
```

Comment:
I see that there are some styles related to the 'md' size that are gone and possibly a new default size of 'lg' added. Whats the storey behind that?

Reply:
Our design seems to fit chakra input size lg where the fontsize is 16px and height is 44px. https://www.chakra-ui.com/docs/components/input#sizes. The default value for input is md. 

In current v2, I don't know why we tried to modified the md input to become 44px (heigh: 11) where we can use the 'lg' directly.